cmd/ebrelayer/txs/parser_test.go

```diff
@@ -0,0 +1,60 @@
+package txs
+
+import (
+	"fmt"
+	"math/big"
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/stretchr/testify/require"
+	"github.com/swishlabsco/cosmos-ethereum-bridge/cmd/ebrelayer/events"
+)
+
+var TestValidator sdk.ValAddress
+var TestEventData events.LockEvent
+
+func init() {
+
+	// Set up testing parameters for the parser
+	testValidator, err := sdk.ValAddressFromBech32("cosmos1xdp5tvt7lxh8rf9xx07wy2xlagzhq24ha48xtq")
+	if err != nil {
+		fmt.Errorf("%s", err)
```

Comment:
```suggestion
		panic(err)
```

Reply:
as you're setting a hardcoded address, you should panic if it's set incorrectly

Reply:
fixed in next push

Reply:
https://github.com/cosmos/peggy/commit/00a712a8055cc31b77c95275ccdd55fdfd19f4ef